extension/src/status.ts

```diff
@@ -34,34 +40,40 @@ export class Status extends Disposable {
     }
   }
 
-  public setAvailability = (available: boolean) => {
+  public async setAvailability(available: boolean) {
     this.available = available
+    await this.config.isReady()
+    const { text, tooltip } = this.getEnvDetails()
+
+    this.textSuffix = text ? `DVC ${text}` : 'DVC'
+
+    this.statusBarItem.tooltip = tooltip
     this.setState(!!this.workers)
   }
 
-  private getText = (isWorking: boolean) => {
```

Comment:
[F] I have no idea why these were arrow functions, obviously haven't been in here in a while.